src/constants.jl

```diff
@@ -5,7 +5,7 @@ const PE = PlanetaryEphemeris
 const src_path = dirname(pathof(PlanetaryEphemeris))
 
 # Integration parameters
-const order = 30
+const order = 25
```

Comment:
Is this change needed?

Reply:
All integrations that we have done with PE are with 25 order polynomias, so I think this is more consistent.